dds_stress_rigSup/start_or_load.st

```diff
@@ -0,0 +1,335 @@
+program start_or_load("P")
+
+#include "ibexSeqPVmacros.h"
+%% #include "seq_snc.h"
+%% #include "epicsTime.h"
+%% #include "string.h"
+%% #include "math.h"
+%% #include "errlog.h"
+%% #include "alarm.h"
+
+option +d;
+option +r;
+option +s;
+
+
+/* Debugging: current state of this state machine */
+PV(string, statemachine_state, "{P}STATEMACHINE:STATE", NoMon);
+
+PV(int, stopMeasurement, "{P}MEAS:STOP:SP", NoMon);
+PV(int, setChannel, "{P}CHANNEL:SP", NoMon);
+PV(int, getChannel, "{P}CHANNEL", Monitor);
+PV(int, setUpLimitEnabled, "{P}ULIMIT:ENABLED:SP", NoMon);
+PV(int, getUpLimitEnabled, "{P}ULIMIT:ENABLED", Monitor);
+PV(int, setLowLimitEnabled, "{P}LLIMIT:ENABLED:SP", NoMon);
+PV(int, getLowLimitEnabled, "{P}LLIMIT:ENABLED", Monitor);
+PV(int, setMotorMode, "{P}MOT:MODE:SP", NoMon);
+PV(int, getMotorMode, "{P}MOT:MODE", Monitor);
+PV(int, setModeStat, "{P}MODE:STAT:SP", NoMon);
+PV(int, getModeStat, "{P}MODE:STAT", Monitor);
+PV(float, target, "{P}TRGT", Monitor);
+PV(float, elong, "{P}ELONG", Monitor);
+PV(float, load, "{P}LOAD:SP", Monitor);
+PV(float, upperLimit, "{P}UPPER_LIMIT:SP", NoMon);
+PV(float, lowerLimit, "{P}LOWER_LIMIT:SP", NoMon);
+PV(int, startMeasurement, "{P}MEAS:START:SP", NoMon);
+PV(int, start, "{P}START.RVAL", Monitor);
+PV(int, setLoad, "{P}SET_LOAD.RVAL", Monitor);
+PV(int, setSpeed, "{P}CHANGE_SPEED.RVAL", Monitor);
+PV(int, setSpeedMode, "{P}MODE:SPEED:SP", NoMon);
+PV(int, setControl, "{P}CONTROL:SPEED:SP", NoMon);
+PV(int, setDirection, "{P}DIRECTION:SPEED:SP", NoMon);
+PV(int, getSpeedMode, "{P}MODE:SPEED", Monitor);
+PV(int, getControl, "{P}CONTROL:SPEED", Monitor);
+PV(int, getDirection, "{P}DIRECTION:SPEED", Monitor);
+PV(int, speedVal, "{P}_SPEED:SP", NoMon);
+PV(int, setSpeedVal, "{P}SPEED:SP", Monitor);
+PV(int, getSpeedVal, "{P}_SPEED", Monitor);
+PV(int, setSpeedSP, "{P}SPEED:SET:SP", NoMon);
```

Comment:
should the speeds be `float`  - epics records look like they are ai/ao?

Reply:
ah, they should (except for speed:set:sp